htdocs/accountancy/class/accountingjournal.class.php

```diff
@@ -1430,7 +1438,7 @@ public function writeIntoBookkeeping(User $user, &$journal_data = array(), $max_
 							$bookkeeping->subledger_label = $line['subledger_label'];
 							$bookkeeping->numero_compte = $line['numero_compte'];
 							$bookkeeping->label_compte = $line['label_compte'];
-							$bookkeeping->label_operation = $line['label_operation'];
+							$bookkeeping->label_operation = dol_string_nohtmltag($line['label_operation']);
```

Comment:
$bookkeeping->label_operation is a data in memory. We should not have a formatting function on it.
If we use the data later as html output, the html escaping must be done when the rendering is done, not in assignement in memory.